### PR TITLE
test1-2 commit

### PR DESCRIPTION
**Pull request by bbowl**
Added tests:
- Test 1 - Perform Login
- Test 2 - Add product to the cart

### CODE REVIEW

File: tests/test1-2.spec.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  await expect(page.locator("//div[@data-test='inventory-item'][1]")).toBeVisible();
+  // Product 2 is displayed after login
+  await expect(page.locator("//div[@data-test='inventory-item'][2]")).toBeVisible();
+});
+
+test('test2_add_2_items_to_shopping_cart', async ({ page }) => {
+  //Add first item it the product list
+  await page.locator('[data-test^="add-to-cart"]').nth(0).click();
+  //Check that Shopping cart badge shows "1"
+  await expect(page.locator("//span[@data-test='shopping-cart-badge'][text()='1']")).toBeVisible();
```
Comment (hlazkov) [inline tests/test1-2.spec.js:25]: 1. Its better to check if `shopping-cart-badge` has text `1` explicitly
2. Its better to use CSS or xpath selectors through whole project, not mixing it.

File: tests/test1-2.spec.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  // Navigate to Shopping Cart List
+  await page.locator('[data-test="shopping-cart-link"]').click();
+  // Check that item is shown on the Shopping Cart list
+  await expect(page.locator("//div[@data-test='inventory-item'][1]")).toBeVisible();
```
Comment (hlazkov) [inline tests/test1-2.spec.js:29]: Same here, lets use CSS selector instead

**Review by hlazkov — CHANGES_REQUESTED** [review]
Looks nice, but recheck on comments please!

**Comment by AlinaZhykharieva** [comment]
great job! that is nice you`ve changed all locators to Xpath, but here almost always we have class or id for element, so CSS will be preferable from my POV. But it is your decision if you will have time you can try it with CSS.

**Review by AlinaZhykharieva — APPROVED** [review]

**Pull request merged**